src/pages/contribute.js

```diff
@@ -174,6 +194,48 @@ export default function Home () {
             </div>
           </div>
         </div>
+        <Modal isShown={showMoreModal} onClose={() => { setShowMoreModal(false) }} title={selectedTopic}>
+          {(selectedTopic === 'TechLadies Bootcamp') && (
+            <>
+              Help create our most impactful program! We are looking for <b>3 - 4 Bootcamp Leads, 8 - 10 Coaches, 1 Tech Lead, and 1 Product Lead.</b>
+
+              <br />
+              <br />
+
+              <h4>Bootcamp Leads</h4>
+              Behind the magic of our most impactful program are Bootcamp Leads who plan and execute this program, including sourcing for NGOs, recruiting coaches, marketing, and running the bootcamp (including pre-bootcamp workshops). It’s time-consuming but very rewarding. Don’t worry, we have a playbook ready.
+
+              <br />
+              <br />
+              <p>
+                <b><i>Ideal profile:</i></b> You have good project management skills and can commit ~8 hours per month in 2021.
+              </p>
+
+              <h4>Coaches</h4>
+              Pair programming with participants
+
+              <br />
+              <br />
+              <p>
+                <b><i>Ideal profile:</i></b> Technical
+              </p>
+
+              <a className="btn btn-primary" href="https://docs.google.com/forms/d/e/1FAIpQLSepj-kx0qMTe7stfAiRB9slYA-HvTTiT3GvbodLj23geH8ZIg/viewform">
+                Volunteer for TechLadies Bootcamp
+              </a>
+            </>
+
+          )}
+          {(selectedTopic === 'TechLadies Mentorship') && (
+            <div>Filler</div>
+          )}
+          {(selectedTopic === 'TechLadies Meet') && (
+            <div>Filler</div>
+          )}
+          {(selectedTopic === 'TechLadies Community') && (
```

Comment:
to minimize error, make it sth like `community` , `meet`, `mentorship`, `bootcamp`